glob_fun.py

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env python
+# HACK A TRONIC SPIKE
+#
+# Playing with a simplistic mechanism for controlling what to include and what to exclude.
+# Globs are annoyingly tricky at times, but I like having some kind of pattern.
+#
+# No command line here, all "parameters" are environment variables.
+# Simple use:
+#     % IN_DIR=my_dir ./glob_fun.py
+# Watch the fun:
+#     % TRACE=1 IN_DIR=my_dir ./glob_fun.py
+
+import os
+from pathlib import Path
+
+# HACK: No config file, fake it out here, this is just a spike
+DIRECTIVES = '''
+Include: .
+Exclude: *.json
+Exclude: workspace*
+Include: snippets
+Exclude: plugins
+Include: plugins/buttons
+Exclude: plugins/buttons/styles.css
+Include: plugins/tag-wrangler
+'''
```

Comment:
This sample list is to show exluding a tree and then including parts (plugins/buttons) and then excluding parts of that. For a real set of directives, there is no point in excluding plugins and then re-including buttons and tag-wrangler, those would just be straight up includes.

This is of course based on my own test vault.
If we decide to go this route, I would add a self-test for this part of the code.